ibis/backends/dask/execution/selection.py

```diff
@@ -199,3 +198,59 @@ def execute_selection_dataframe(
 
     # drop every temporary column we created for ordering or grouping
     return result.drop(temporary_columns, axis=1)
+
+
+def _compute_predicates(
```

Comment:
Why do you need to add this?

Reply:
It directly calls execute, so if we don't copy it over, then it uses the pandas dispatcher.

Reply:
I think if we do sth like https://github.com/ibis-project/ibis/pull/2606/files/f140d3a4985b67b65ebc1542c2a51658fb93928f#r569702202

We can keep most of the "skeletons" shared between pandas/dask and plugin their own "dispatchers"

Reply:
Can be a follow up though